javascript/node/selenium-webdriver/safari.js

```diff
@@ -200,14 +217,19 @@ class Driver extends webdriver.WebDriver {
    * @return {!Driver} A new driver instance.
    */
   static createSession(opt_config, opt_flow) {
-    let caps;
+    let caps, exe;
+
     if (opt_config instanceof Options) {
       caps = opt_config.toCapabilities();
     } else {
-      caps = opt_config || Capabilities.safari()
+      caps = opt_config || Capabilities.safari();
+    }
+
+    if (caps.get(TECHNOLOGY_PREVIEW_CAPABILITY_KEY)) {
```

Comment:
This will always be false. You don't set the capability on the top-level capabilities object.